build-tools/automation/yaml-templates/variables.yaml

```diff
@@ -56,3 +56,6 @@ variables:
   value: 'cat != SystemApplication & cat != TimeZoneInfo & cat != Localization'
 - name: RunMAUITestJob
   value: true
+# Workaround: https://github.com/dotnet/msbuild/issues/10378
+- name: MSBUILDDISABLEFEATURESFROMVERSION
```

Comment:
looks like they reverted the PR which caused this. Should we wait?

Reply:
I think it's fine to merge this sooner if I get it green.

The next bump, I'll try to take it out.